json_journal/data_analysis.py

```diff
@@ -82,3 +85,74 @@ def total_profit(full_account_name_balances):
             pass
     total = (income * (-1)) - expenses
     return total
+
+
+def calculate_profit(ledger):
+    '''
+    Calculate the profit of a ledger file.
+
+    Parameters
+    ----------
+    Ledger : list of dicts
+        Loaded json file.
```

Comment:
Could be a loaded json file. Doesn't have to be though.

Reply:
done